build.gradle

```diff
@@ -29,7 +29,7 @@ buildscript {
     errorProneJavac: 'com.google.errorprone:javac:9+181-r4173-1',
     junit: 'junit:junit:4.13.2',
     truth: 'com.google.truth:truth:1.1.3',
-    robolectric: 'org.robolectric:robolectric:4.4',
+    robolectric: 'org.robolectric:robolectric:4.6.1',
```

Comment:
Bitmap transformations weren't copying BitmapConfig over.  Bumping Robolectric fixed that.